wikimedia/wikipedia-ios
# [YiR] Add localizations for it and es

**Phabricator:** https://phabricator.wikimedia.org/T376666

### Notes
* Add localized strings for IT and ES

### Test Steps
1. Set the device region to a valid one
2. Set device language to Italian and then Spanish
3. Verify copy and interface translations
